app/assets/v2/js/pages/leaderboard.js

```diff
@@ -1,10 +1,17 @@
-$(document).ready(function(){
-    $(".leaderboard_entry").each(function(){
-        $(this).changeElementType('a'); // hack so that users can right click on the element
```

Comment:
i'd still like for these items to be `a` tags because that means that a user can right click on them

Reply:
is clickable but not using Anchor tag. It would need a lot of css fix in order to use the `<a>` tag. See comment bellow.

Reply:
ok, ill make the change after merge